### PR TITLE
New logo

### PR DESCRIPTION
**Pull request by JGeiser9**
This PR replaces the placeholder 'H' logo with the new logo we decided on. I removed unused SVG logos, and updated some styling to stay consistent with the prior HNScan version.

### CODE REVIEW

File: src/components/layout/Navigation.js [hunk 1]
```diff
@@ -62,7 +63,7 @@ export default function Navigation() {
   const [moreDropdownActive, updateMoreDropdownActive] = useState(false);
   const [toolsDropdownActive, updateToolsDropdownActive] = useState(false);
   return (
-    <BorderedNav height={"60px"}>
+    <BorderedNav>
```
Comment (shanejann) [inline src/components/layout/Navigation.js]: 🤔 Was that height prop even doing anything? I'd expect we would have had to do something like `style={height: 60px}` OR take in the `prop` within the styled component, but we were doing neither of those! Anyway, nice catch!

File: src/components/styles/NavBar/NavBar.js [hunk 2]
```diff
@@ -327,7 +327,7 @@ color: ${props => props.theme.global.textColor};
 `;
 
 const Brand = styled.div`
-  align-items: stretch;
+  align-items: center;
```
Comment (shanejann) [inline src/components/styles/NavBar/NavBar.js]: Honestly, I'm a tad curious at the difference b/w these two. I should play around with them...

Comment (shanejann) [reply src/components/styles/NavBar/NavBar.js]: Refreshing myself here https://css-tricks.com/snippets/css/a-guide-to-flexbox/ It kinda just seems like the stretch is the same as center, the only difference being that stretch fills up the container it's in. 🤷‍♂

File: src/components/layout/Navigation.js (hunk 1 above)
Comment (kilpatty) [reply src/components/layout/Navigation.js]: We do -> https://github.com/HandshakeAlliance/HNScan/blob/staging/src/components/styles/NavBar/NavBar.js#L6

this just mimics the behavior the BulmaReact library had, but the default value is 3.25em which is (3.25 * 16) == ~52. So roughly the same height. In this particular scenario its a non-change bc it sets min-height and not height, so with the logo being larger than 52px it stays the same.

File: src/components/styles/NavBar/NavBar.js (hunk 2 above)
Comment (kilpatty) [reply src/components/styles/NavBar/NavBar.js]: I'd keep this as is imo, it's from bulimia, and I think on "near-tablets" having the nav spread out across the space is easier to navigate with.

File: src/components/layout/Navigation.js (hunk 1 above)
Comment (shanejann) [reply src/components/layout/Navigation.js]: oh nice eye. I didn't realize BorderedNav was an extension of NavBar. Makes sense 👍

Comment (JGeiser9) [reply src/components/layout/Navigation.js]: Oops, didn't realized this was being used as a prop. Reverting back to the original in my next commit

File: src/components/styles/NavBar/NavBar.js (hunk 2 above)
Comment (JGeiser9) [reply src/components/styles/NavBar/NavBar.js]: Fixed this with recent commit. There was an issue with the nav not accepting the props for height

**Pull request merged**